plugins/inputs/win_eventlog/util.go

```diff
@@ -51,6 +51,7 @@ func GetFromSnapProcess(pid uint32) (uint32, uint32, string, error) {
 	}
 	defer windows.CloseHandle(snap)
 	var pe32 windows.ProcessEntry32
+	//nolint:gosec // G103: Use of unsafe calls should be audited
 	pe32.Size = uint32(unsafe.Sizeof(pe32))
```

Comment:
I think this is a valid use to determine the size of that struct. So how about
```suggestion
	//nolint:gosec // Valid use of unsafe calls to determine the size of the struct
	pe32.Size = uint32(unsafe.Sizeof(pe32))
```

Reply:
Changed.